examples/rpc/index.ts

```diff
@@ -0,0 +1,174 @@
+import { RemoteParticipant, Room, RoomEvent, RpcRequest } from '@livekit/rtc-node';
+import { RPC_ERROR_UNSUPPORTED_METHOD } from '@livekit/rtc-node/dist/rpc';
+import { randomBytes } from 'crypto';
+import { config } from 'dotenv';
+import { AccessToken } from 'livekit-server-sdk';
+
+config({ path: '.env.local', override: false });
+const LIVEKIT_API_KEY = process.env.LIVEKIT_API_KEY;
+const LIVEKIT_API_SECRET = process.env.LIVEKIT_API_SECRET;
+const LIVEKIT_URL = process.env.LIVEKIT_URL;
+if (!LIVEKIT_API_KEY || !LIVEKIT_API_SECRET || !LIVEKIT_URL) {
+  throw new Error('Missing required environment variables. Please check your .env.local file.');
+}
+
+async function main() {
+  const roomName = `rpc-test-${randomBytes(4).toString('hex')}`;
+
+  console.log(`Connecting participants to room: ${roomName}`);
+
+  const [requestersRoom, greetersRoom, mathGeniusRoom] = await Promise.all([
+    connectParticipant('requester', roomName),
+    connectParticipant('greeter', roomName),
+    connectParticipant('math-genius', roomName),
+  ]);
+
+  // Register all methods for the receiving participant
+  await registerReceiverMethods(greetersRoom, mathGeniusRoom);
+
+  try {
+    console.log('\n\nRunning greeting example...');
+    await Promise.all([performGreeting(requestersRoom)]);
+  } catch (error) {
+    console.error('Error:', error);
+  }
+
+  try {
+    console.log('\n\nRunning math example...');
+    await Promise.all([
+      performSquareRoot(requestersRoom)
+        .then(() => new Promise<void>((resolve) => setTimeout(resolve, 2000)))
+        .then(() => performQuantumHypergeometricSeries(requestersRoom)),
+    ]);
+  } catch (error) {
+    console.error('Error:', error);
+  }
+
+  console.log('\n\nParticipants done, disconnecting...');
+  await requestersRoom.disconnect();
+  await greetersRoom.disconnect();
+  await mathGeniusRoom.disconnect();
+
+  console.log('Participants disconnected. Example completed.');
+
+  process.exit(0);
+}
+
+const registerReceiverMethods = async (greetersRoom: Room, mathGeniusRoom: Room): Promise<void> => {
+  greetersRoom.localParticipant?.registerRpcMethod(
+    'arrival',
+    async (request: RpcRequest, sender: RemoteParticipant) => {
+      console.log(`[Greeter] Oh ${sender.identity} arrived and said "${request.payload}"`);
+      await new Promise((resolve) => setTimeout(resolve, 2000));
+      return "Welcome and have a wonderful day!";
+    },
```

Comment:
seems like a dead simple way to declare support for an RPC method? you can throw in here too